src/components/LoginForm.tsx

```diff
@@ -32,41 +33,38 @@ export const LoginForm: React.FC<LoginFormProps> = ({
 
     setLoading(true);
 
+    // code below commented out because new test user and password are now being tested in the backend. ~Bree-Mass
```

Comment:
We can remove this code. The good thing about `git` is we can always view history if we ever need to restore code. But for now, we don't need this.